demisto_sdk/commands/validate/validators/ST_validators/ST111_no_exclusions_schema.py

```diff
@@ -33,9 +33,23 @@ def obtain_invalid_content_items(
         return invalid_content_items
 
     def is_missing_section_fields(self, content_item: ContentTypes) -> str:
-        section_order = content_item.data.get("sectionorder") or content_item.data.get(
-            "sectionOrder"
-        )
+        """
+        Validates section-related fields in content items.
+
+        Returns specific error if 'sectionOrder' (camelCase) is used instead of 'sectionorder' (lowercase).
+        Returns missing field error if neither 'sectionorder' nor 'sectionOrder' exists.
+        Validates that all configuration parameters have required section fields.
+
+        Returns:
+            str: Error message if validation fails, empty string if validation passes.
+        """
+        section_order_lowercase = content_item.data.get("sectionorder")
```

Comment:
As discussed offline, I think it is better to check if we have the key "sectionorder" in this exact form (lowercase), and if not - to shout at the user. 
You can mention in the error that it should be lower case, but no need to search specifically for "sectionOrder", because a user could add for example, "SectionOrder".